api-server/src/logging/ExceptionLogger.ts

```diff
@@ -13,6 +13,8 @@ export class ExceptionLogger extends BaseExceptionFilter {
       : 'unknown method';
 
     console.info(`Handled Exception (${url}, ${method}): "${message}"`);
+    console.error(exception);
+
```

Comment:
sneak sneaked this in here. This will make sure when there's an exception we get a full stack trace along with the basic message. I think this will be useful for debugging production issues.

i at least want to experiment with having this on. If it gets too noise we can take it back